serine.galstyan/repo/Lesson_3/exercise_13/exercise_13.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/evn python3
+
+meal_cost = int(input("Enter the cost of the meal: "))
+tax_rate = 0.09
```

Comment:
CSW: Constant value

Reply:
The task requires to compute the tax and tip for the meal.